src/averageAges.js

```diff
@@ -15,11 +15,23 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let manList;
+
+  if (century) {
+    manList = people.filter(
+      person => person.sex === 'm' && Math.ceil(person.died / 100) === century,
+    );
+  } else {
+    manList = people.filter(
+      person => person.sex === 'm',
+    );
+  }
```

Comment:
`if else` is forbidden in this task. Try to use [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator#syntax)